### DISCUSSION TITLE
The "jquery-validate" Snyk vulnerability report could point to the right library

### DISCUSSION THREAD

**Discussion by mgroetan** [cdnjs/cdnjs#14137] [General]
This library is deprecated on npm, but exists under another name: "jquery-validation".
This is also reflected in the URL to its GitHub repo.
The link to the Snyk vulnerability scan report should also be updated accordingly.

**Comment by MattIPv4** [comment]
👋 The Snyk integration on the site pulls the package name based on the cdnjs library name, or the NPM package the library is set to auto-update from. As in this case the library is set to auto-update from a git repository, there is no NPM package configured, and so the library name is used for Snyk.

You're welcome to open an issue in cdnjs/static-website with a suggestion on how the logic could be refactored for this to generate a more accurate Snyk URL, or an issue could be opened in cdnjs/tools to add support for a new package property to manually define the Snyk package name.
